### ISSUE TITLE
Is this repo no longer maintained?

### ISSUE THREAD

**Issue by ffxsam** [eclipse/paho.mqtt.javascript#241]
@jpwsutton @knolleary Would you mind adding more people to the contributors list? There are some issues that need to be fixed, but it won't matter if we can't get our PRs merged and published.

**Comment by ToshioMagic** [comment]
Agreed. I've banged my head on my desk for 3 days before I found out this isn't maintained.

**Comment by CarlosBalladares** [comment]
any alternatives for the browser?

**Comment by nandlab** [comment]
> any alternatives for the browser?

There is also [MQTT.js](https://github.com/mqttjs/MQTT.js).

**Comment by davidhmays** [comment]
I'm struggling to get type definitions for the browser version of MQTT.js. 
The NPM version has working type definitions, but won't work in the browser as it uses some things like process.env that only work in a Node environment.

**Comment by robertsLando** [comment]
MQTTjs has full typescript support starting from version 5 and there have been many improvements on browser side in latest versions. see https://github.com/mqttjs/MQTT.js/releases
